spec/student_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../students'
+require_relative '../persons'
+
+describe Student do
+  context 'with an input of names and ages' do
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/217907181-9d5d6864-46ef-4c86-b680-de631af198c8.png)

- [ ] You didn't test **play_hooky** method :fearful: Kindly test this method to test your student class fully :+1: